tests/Metadata/Driver/YamlDriverTest.php

```diff
@@ -103,6 +103,9 @@ public function testLoadingMultipleMetadataExtensions(): void
         );
     }
 
+    /**
+     * @return YamlDriver
```

Comment:
Can't we inline it to the method signature?

Reply:
Not possible in PHP7.2 that is still supported - needs to be compatible with parent class.